src/firestore/indexes.ts

```diff
@@ -563,11 +577,16 @@ export class FirestoreIndexes {
       clc.yellow(parsedName.fieldPath) +
       "] --";
 
-    field.indexConfig.indexes.forEach((index) => {
-      const firstField = index.fields[0];
-      const mode = firstField.order || firstField.arrayConfig;
-      result += " (" + mode + ")";
-    });
+    const fieldIndexes = field.indexConfig.indexes || [];
+    if (fieldIndexes.length > 0) {
+      fieldIndexes.forEach((index) => {
+        const firstField = index.fields[0];
+        const mode = firstField.order || firstField.arrayConfig;
+        result += " (" + mode + ")";
```

Comment:
prefer template strings:

```suggestion
        result += ` (${mode})`;
```